golem-cli/src/cloud.rs

```diff
@@ -12,10 +12,47 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-pub mod auth;
-pub mod clients;
-pub mod command;
-pub mod factory;
-pub mod main;
-pub mod model;
-pub mod service;
+use chrono::{DateTime, Utc};
+use derive_more::{Display, FromStr, Into};
+use serde::{Deserialize, Serialize};
+use std::fmt::{Debug, Display, Formatter};
+use uuid::Uuid;
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct CloudAuthenticationConfig {
```

Comment:
what is the reason, why those configuration remaining in OSS?

profiles?

Reply:
Yes. It is for shared profiles. I can hide it with `serde_json::Value`, but there is no good reason for this.